notebooks/SMAP/3.0 SMAP Quality Flags.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ret_flags = {
</pre><p>This should be retrieved from the Dataset attributes.  Note that the flag meanings given in this notebook are different from those in the attributes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>